python/ray/_raylet.pyx

```diff
@@ -845,6 +847,15 @@ cdef void terminate_asyncio_thread() nogil:
         core_worker.destroy_event_loop_if_exists()
 
 
+def connect_to_gcs(ip, port, password):
+    print("Connecting to gcs: ", ip, port, password)
```

Comment:
We should use the logger instead of plain print statements.

```suggestion
    logger.info(f"Connecting to gcs: {ip}, {port}, {password}")
```

Reply:
Oh, I actually need to remove it. It's debugging code. It'll print in cpp side for connecting I think.